### PR TITLE
SM-33 navigation

### PR DESCRIPTION
**Pull request by n0macx**
`Nav styled with simple logic. 

image
image
image

### CODE REVIEW

File: src/features/layout/nav/NavBar.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+const NavBar = (prop: {
+  t: any;
+  className: string;
+  handleNav?: () => void;
+}) => {
+  return (
+    <>
+      {[
+        [`${prop.t('products')}`, '/product-series'],
```
Comment (vieraboschkova) [inline src/features/layout/nav/NavBar.tsx]: Is destructuring t an option?

File: src/features/layout/nav/NavBar.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import { NavLink } from 'react-router-dom';
+
+const NavBar = (prop: {
+  t: any;
+  className: string;
+  handleNav?: () => void;
+}) => {
+  return (
+    <>
+      {[
```
Comment (vieraboschkova) [inline src/features/layout/nav/NavBar.tsx]: You could define this array outside of the return statement for cleaner code

File: src/features/layout/nav/NavBar.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+          to={link}
+          key={title}
+          onClick={prop.handleNav}
+          className={({ isActive }) =>
+            isActive
+              ? `${prop.className} md:text-lg sm:text-base font-medium`
+              : prop.className
+          }
+        >
+          {title}
```
Comment (vieraboschkova) [inline src/features/layout/nav/NavBar.tsx]: Perhaps you could translate it here if possilbe, then the array could be just [products, about, service, contact], and the routes could be set accordingly

File: src/features/layout/nav/Nav.tsx [hunk 4]
```diff
@@ -1,24 +1,100 @@
...
-      <NavLink to="/user"> UserPage </NavLink>
-      <NavLink to="/product-series"> ProductSeriesPage </NavLink>
-      <NavLink to="/admin"> AdminPage </NavLink>
+    <div className="flex justify-between items-center h-20 sm:h-36 md:h-44 max-w-7xl mx-auto sm:px-3 md:px-6 lg:px-8">
+      <div
+        className={
+          !nav ? 'p-6 sm:p-3 lg:p-4 w-36 sm:w-48  md:w-64 lg:w-72' : 'hidden'
+        }
+      >
+        <img src={logo} alt="smallets logo" width="100%" height="100%" />
```
Comment (vieraboschkova) [inline src/features/layout/nav/Nav.tsx]: Logo could be a link to a landing page perhaps?

File: src/features/layout/nav/Nav.tsx [hunk 5]
```diff
@@ -1,24 +1,100 @@
...
+      </div>
+      <section className="hidden sm:block">
+        <div className="flex p-4 pl-8 gap-8 justify-end text-2xl">
+          <NavLink to="/login">
+            <BiUser />
+          </NavLink>
+          <NavLink to="/cart">
+            <BsCart3 />
+          </NavLink>
```
Comment (vieraboschkova) [inline src/features/layout/nav/Nav.tsx]: These two links, and the next row of links, change the location of the nav links, they jump a bit (to reproduce: click on cart and then on products, and then on cart again. the text changes position

Comment (n0macx) [reply src/features/layout/nav/Nav.tsx]: Fixed. Thanks for the comment.

File: src/features/layout/nav/Nav.tsx (hunk 4 above)
Comment (n0macx) [reply src/features/layout/nav/Nav.tsx]: Good point. Added. Thanks.

File: src/features/layout/nav/NavBar.tsx (hunk 3 above)
Comment (n0macx) [reply src/features/layout/nav/NavBar.tsx]: Yes it looks a lot better now. Thanks!

File: src/features/layout/nav/NavBar.tsx (hunk 2 above)
Comment (n0macx) [reply src/features/layout/nav/NavBar.tsx]: Done.

File: src/features/layout/nav/NavBar.tsx (hunk 1 above)
Comment (n0macx) [reply src/features/layout/nav/NavBar.tsx]: Refactored.

**Review by KatarzynaChmielecka — APPROVED** [review]
Awesome :)

**Pull request merged**